app/web/src/newhotness/layout_components/AttributeInput.vue

```diff
@@ -586,7 +586,7 @@
       <CodeEditorModal
         ref="codeEditorModalRef"
         :title="`Set Value For ${displayName}`"
-        :codeEditorId="`${displayName}-${changeSetId}-${prop?.id}`"
```

Comment:
code editor actually pulls in the change set store to prepend the changeSetId to the key, so this isn't needed